internal/storage/storage_handle_test.go

```diff
@@ -100,3 +101,10 @@ func (t *StorageHandleTest) TestNewStorageHandleWithZeroMaxConnsPerHost() {
 
 	t.invokeAndVerifyStorageHandle(sc)
 }
+
+func (t *StorageHandleTest) TestNewStorageHandleWithDifferentUserAgent() {
+	sc := getDefaultStorageClientConfig()
+	sc.UserAgent = "UserAgent"
```

Comment:
Same: Can we set some close to real-world UserAgent ?

Reply:
fixed